connector/src/main/resources/schema/wildfly-datasources_7_0.xsd

```diff
@@ -908,14 +908,26 @@
                 </xs:documentation>
               </xs:annotation>
             </xs:element>
-        </xs:sequence>
-        <xs:element name="credential-reference" type="credential-reference:credentialReferenceType" minOccurs="0">
-            <xs:annotation>
-              <xs:documentation>
-                Credential to be used by the configuration.
-              </xs:documentation>
-            </xs:annotation>
-          </xs:element>
+          </xs:sequence>
+          <xs:sequence>
+            <xs:element name="user-name" type="xs:token" minOccurs="0">
+              <xs:annotation>
+                <xs:documentation>
+                  <![CDATA[[
+                    Specify the username used when creating a new connection.
+                            Ex: <user-name>sa</user-name>
+                  ]]>
+                </xs:documentation>
+              </xs:annotation>
+            </xs:element>
```

Comment:
Rather than repeat the user-name element: (user-name, password) | (user-name, credential-reference)
why not use: (user-name, (password | credential-reference))